### PR TITLE
Update ios-install.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed the wordings as it is connect VPN and Not establish VPN anymore

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8583

File: microsoft-365/security/defender-endpoint/ios-install.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ This step simplifies the onboarding process by setting up the VPN profile. For a
     - VPN identifier = com.microsoft.scmx
     - In the key-value pairs, enter the key **AutoOnboard** and set the value to **True**.
     - Type of Automatic VPN = On-demand VPN
-    - Click **Add** for **On Demand Rules** and select **I want to do the following = Establish VPN**, **I want to restrict to = All domains**.
+    - Click **Add** for **On Demand Rules** and select **I want to do the following = Connec VPN**, **I want to restrict to = All domains**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install.md]:
```suggestion
    - Select **Add** for **On Demand Rules** and select **I want to do the following = Connect VPN**, **I want to restrict to = All domains**.
```

File: microsoft-365/security/defender-endpoint/ios-install.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ Admins can configure Microsoft Defender for Endpoint to deploy and activate sile
     - VPN identifier = com.microsoft.scmx
     - In the key-value pairs, enter the key **SilentOnboard** and set the value to **True**.
     - Type of Automatic VPN = On-demand VPN
-    - Select **Add** for **On Demand Rules** and select **I want to do the following = Establish VPN**, **I want to restrict to = All domains**.
+    - Select **Add** for **On Demand Rules** and select **I want to do the following = Connec VPN**, **I want to restrict to = All domains**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install.md]:
```suggestion
    - Select **Add** for **On Demand Rules** and select **I want to do the following = Connect VPN**, **I want to restrict to = All domains**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Hi @msbemba, you have missed one suggested change. Can you also commit that, please?

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**